components/pkg-export-helm/src/main.rs

```diff
@@ -84,17 +85,30 @@ fn cli<'a, 'b>() -> clap::App<'a, 'b> {
         .arg(
             Arg::with_name("VERSION")
                 .value_name("VERSION")
+                .short("v")
                 .long("version")
                 .validator(valid_version)
-                // Keep the default version here in sync with chartfile::DEFAULT_VERSION
-                .help("Version of the chart to create (default: 0.0.1)"),
+                .help("Version of the chart to create")
+                .default_value(chartfile::DEFAULT_VERSION),
         )
         .arg(
             Arg::with_name("DESCRIPTION")
                 .value_name("DESCRIPTION")
                 .long("desc")
                 .help("A single-sentence description"),
         )
+        .arg(
+            Arg::with_name("OPERATOR_VERSION")
+                .value_name("OPERATOR_VERSION")
+                .long("operator-version")
+                .validator(valid_version)
+                .help("Version of the Habitat operator to set as dependency")
+                .default_value(deps::DEFAULT_OPERATOR_VERSION),
+        )
+        .arg(Arg::with_name("DOWNLOAD_DEPS").long("download-deps").help(
+            "Whether to download dependencies. Currently only the Habitat Kubernetes Operator \
```

Comment:
Nit: a bit weird phrasing "Currently only the Habitat Kubernetes Operator". maybe "The Kubernetes habitat-operator is the only dependancy currently."

Reply:
ok